hooks/pre_gen_project.py

```diff
@@ -72,11 +72,18 @@
     sys.exit(1)
 
 if (
-    "{{ cookiecutter.cloud_provider }}" == "GCP"
-    and "{{ cookiecutter.mail_service }}" == "Amazon SES"
-) or (
-    "{{ cookiecutter.cloud_provider }}" == "None"
-    and "{{ cookiecutter.mail_service }}" == "Amazon SES"
+    (
+        "{{ cookiecutter.cloud_provider }}" == "GCP"
+        and "{{ cookiecutter.mail_service }}" == "Amazon SES"
+    )
+    or (
+        "{{ cookiecutter.cloud_provider }}" == "AZURE"
+        and "{{ cookiecutter.mail_service }}" == "Amazon SES"
+    )
+    or (
+        "{{ cookiecutter.cloud_provider }}" == "None"
+        and "{{ cookiecutter.mail_service }}" == "Amazon SES"
+    )
```

Comment:
Maybe it's time to simplify this?

```suggestion
    (
        "{{ cookiecutter.mail_service }}" == "Amazon SES" 
        and "{{ cookiecutter.cloud_provider }}" != "AWS"
    )
```